op-node/rollup/superchain.go

```diff
@@ -101,8 +101,6 @@ func LoadOPStackRollupConfig(chainID uint64) (*Config, error) {
 		CanyonTime:        chConfig.CanyonTime,
 		DeltaTime:         chConfig.DeltaTime,
 		EcotoneTime:       chConfig.EcotoneTime,
-		// TODO(seolaoh): uncomment this when geth updated
-		// KromaMPTTime:           chConfig.KromaMPTTime,
```

Comment:
Why is this removed? Since setting the genesis block is irrelevant with KromaMPTTime?

Reply:
Because it is not declared in the superchain-registry. Furthermore, we don't use this function.